marklogic-data-hub-central/ui/e2e/cypress/integration/explore/scenarios.json.tsx

```diff
@@ -709,8 +709,7 @@ describe("scenarios for final/staging databases for zero state and explore pages
     browsePage.getFinalDatabaseButton().click();
     browsePage.waitForSpinnerToDisappear();
 
-    cy.contains("Showing 1-20 of 36 results", {timeout: 5000});
-    browsePage.getTotalDocuments().should("be.equal", 36);
+    browsePage.getTotalDocuments().should("be.greaterThan", 30);
```

Comment:
@timur-isangulov I do not think we need to change the expect statement. This is going fine in develop branch. the number difference is due to the additional documents that are getting uploaded in mapping test.

Reply:
Reverted the change. Thanks.